src/parcsr_ls/par_amg_setup.c

```diff
@@ -1757,6 +1792,32 @@ hypre_BoomerAMGSetup( void               *amg_vdata,
                                                 num_functions, dof_func_data, debug_flag,
                                                 agg_P12_trunc_factor, agg_P12_max_elmts, &P1);
                }
+               else if (agg_interp_type == 4 || agg_interp_type == 8 || agg_interp_type == 9)
```

Comment:
I am not sure if this would matter essentially, but, from a superficial level, I guess this change would change the logic of the if-else.... Someone is more familiar with the logic of the if-else branches here, maybe @ulrikeyang , may need look it more carefully. Thanks!

Reply:
I appreciate if Ulrike can double-check. The idea here was to remove duplicated code